Fix: Do not allow empty comments to be posted

sausagroll90 opened the pull request:
Users could previously press the submit button in the comment box and it would send a comment with an empty body. This change fixes that by disabling the submit button until the user has typed something in the comment box.

pippaamy commented on __tests__/app.test.js:
```diff
@@ -394,14 +394,40 @@ describe("/api/articles/:article_id/comments", () => {
+    test("400: when comment_id is invalid", () => {
+      return request(app)
+        .delete("/api/comments/bad_comment")
+        .expect(400)
```
lovely testing!

pippaamy commented on model/comments.model.js:
```diff
@@ -43,3 +43,13 @@ exports.insertComment = async (requestBody, article_id) => {
+exports.removeComment = async (id) => {
+  const { rows } = await db.query(
+    "DELETE FROM comments WHERE comment_id=$1 RETURNING *",
+    [id]
```
this is fab, well done!

pippaamy reviewed (commented):
Great job, this is fab!

sausagroll90 committed:
Fix: Do not allow empty comments to be posted

Users could previously press the submit button in the comment box and it would send a comment with an empty body.
This change fixes that by disabling the submit button until the user has typed something in the comment box.
